app/src/main/java/org/sopt/cgv/core/designsystem/component/card/CompTimeCard.kt

```diff
@@ -0,0 +1,156 @@
+package org.sopt.cgv.core.designsystem.component.card
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.sopt.cgv.core.designsystem.theme.*
+import org.sopt.cgv.R
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun CompTimeCard(
+    modifier: Modifier = Modifier,
+    startTime: String,
+    endTime: String,
+    currentSeats: Int,
+    totalSeats: Int,
+    isMorning: Boolean,
+    isActivated: Boolean = false,
+    onClick: () -> Unit,
+){
+
+
+    val backgroundColor = if (isActivated) White else Gray700
+    val backgroundColor2 = if (isActivated) Gray200 else Gray700
+    val leftSeatsColor = if (isActivated) PrimaryRed400 else Green
```

Comment:
요 부분 
```suggestion
    .background(if (isActivated) White else Gray700)
    
    .background(if (isActivated) Gray200 else Gray700),
   
```
이렇게 주는 거 어떠실지요!

Reply:
이렇게 하니까 쓸데없는 변수 선언도 줄일 수 있고, 가독성도 더 좋아지네요!